File: apps/web/cadastro_de_memes/src/app/model/Meme.ts
export class Meme{
    id: number | null;
    link_foto:string | null;
    nome:string | null;
    constructor(id: number | null, link_foto:string | null, nome:string | null){
        this.id = id;
        this.link_foto = link_foto;
        this.nome = nome;
    }
}

File: apps/backend/cadastrodememes/src/main/java/br/com/marquescleiton/cadastrodememes/entity/Meme.java
package br.com.marquescleiton.cadastrodememes.entity;

import javax.persistence.*;

@Table(name = "tb_meme")
@Entity(name = "Meme")
public class Meme {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id;
    String link_foto;

    String nome;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getLink_foto() {
        return link_foto;
    }

    public void setLink_foto(String link_foto) {
        this.link_foto = link_foto;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }
}
